lxd/db_images.go

```diff
@@ -105,8 +105,17 @@ func dbImageSourceGet(db *sql.DB, imageId int) (int, api.ImageSource, error) {
 
 }
 
-// dbImageGet gets an ImageBaseInfo object from the database.
-// The argument fingerprint will be queried with a LIKE query, means you can
+// Whether an image with the given fingerprint exists.
+func dbImageExists(db *sql.DB, fingerprint string) (exists bool, err error) {
```

Comment:
If your function can't erroneously report `true` although `dbQueryRowScan()` failed when called as:

    exists, _, := dbImageExists()

and you remove the named parameters I'm satisfied. :)